src/FramedlessClient.ts

```diff
@@ -112,13 +112,13 @@ class FramedlessClient {
         }
     }
 
-    public async getView(fileMetadata: FileMetadata, viewLocation: string, chatToken: OmnichannelChatToken | null = null): Promise<Blob> {
+    public async getView(fileMetadata: FileMetadata, viewLocation: string, chatToken: OmnichannelChatToken | null = null, supportedImagesMimeTypes?: string[]): Promise<Blob> {

```

Comment:
Please add `[]` as default value for `supportedImagesMimeTypes` for backward compatibility.

Reply:
done